README.md

```diff
@@ -323,6 +324,47 @@ docker rm sbot
 ./create-sbot
 ```
 
+## kubernetes setup
+
+Yaml config files for Kubernetes are included in this repository at
+`hack/k8s/deployment.yaml`. This has been tested on DigitalOcean. Tweaks may be
```

Comment:
this is no less a hack than anything else, so i reckon you should move the `k8s` folder to the top-level

Reply:
`hack/k8s` or `deploy/k8s` are typical (though there is no real standard) locations for these files.